.github/workflows/ci.yml

```diff
@@ -197,15 +209,13 @@ jobs:
           echo "Unzip complete"
       - name: Set Environment Variables
         id: set-env
-        shell: bash
+        shell: cmd
         run: |
-          echo "C:\msys64\usr\bin;C:\Program Files (x86)\Common Files\Intel\Shared Libraries\bin;" >> $GITHUB_PATH
-          # Disable SPMD testing
-          echo "NO_DIST=1" >> "$GITHUB_ENV"
-          # Determine dpctl support
-          DPCFLAG="0"
-          if [ $(echo ${{ matrix.PYTHON_VERSION }} | grep '${{ env.DPCTL_PY_VERSIONS }}') ]; then DPCFLAG=''; fi
-          echo "DPCFLAG=${DPCFLAG}" >> "$GITHUB_OUTPUT"
+          echo C:\msys64\usr\bin;>> %GITHUB_PATH%
+          echo NO_DIST=YES>> %GITHUB_ENV%
```

Comment:
We're using `NO_DIST=1` elsewhere.

Reply:
changed.

Reply:
Turns out it broke github actions. Had to revert. Why? Because exporting to GITHUB_ENV via a batch file seems very poorly documented/ operates in an unexpected manner.  Would do it in linux, but I need the full path of the coverage rc file from windows, and running a bash script from github actions forces it into a linux environment (another odd behavior).

Reply:
But why would a '1' in this case make such a difference? Or was it about a different change?

Reply:
Something with how github is writing and reading to the GITHUB_ENV file is dropping the 1 when done from a batch script. Quotations or making it a string with "" will keep the quotations too. Valid strings for the input can be found here: https://github.com/intel/scikit-learn-intelex/blob/main/setup.py#L85 so it should be okay.